packages/smart-wallet/src/smartWallet.js

```diff
@@ -64,8 +64,9 @@ const mapToRecord = map => Object.fromEntries(map.entries());
  */
 
 /**
- * @typedef {{ updated: 'offerStatus', status: import('./offers.js').OfferStatus } |
- *  { updated: 'balance'; currentAmount: Amount }
+ * @typedef {{ updated: 'offerStatus', status: import('./offers.js').OfferStatus }
+ *   | { updated: 'balance'; currentAmount: Amount }
+ *   | { updated: 'bridgeStatus'; status: { error: string } }
```

Comment:
This seem like an abstraction leak to mention "bridge"

Reply:
how so? the smartWallet contract relies on a Cosmos bridge. we've even considered renaming this contract to have bridge in the name.

is there something else you'd suggest?

Reply:
I mean that if this is a value that is exposed to the Wallet UI app, does it make sense for it to be aware that the implementation on chain uses something called a "bridge". The way I see it, it should just know it's sending a "wallet spend action", and how that's routed internally is not its concern. This effectively reifies that implementation detail.

Reply:
Does `executeOffer` update `offerStatus` in all cases of rejections? If so, I'd be willing to leave the unhandled rejection for the default case.

Reply:
Deferring to https://github.com/Agoric/agoric-sdk/issues/7004